server/src/messageHandler.mjs

```diff
@@ -47,6 +48,31 @@ async function handleMessage(message, userId, ws) {
     return;
   }
 
+  //bypass JSON-RPC calls and directly target device. 
+  if(process.env.proxy) {
+    let params = oMsg.params
+    const payload = {
```

Comment:
Is there a reason for re-creating the message and not passing it on as-is?

Reply:
Addressed. Using as is